dashboard/app/controllers/pd/workshop_daily_survey_controller.rb

```diff
@@ -64,16 +64,14 @@ def new_pre_foorm
     end
 
     # General post-workshop survey using foorm system.
-    # GET '/pd/workshop_post_survey?enrollmentCode=code'
     # Enrollment code is an optional parameter, otherwise will show most recent workshop.
     #
     # If the post-survey has been already completed, will redirect to thanks page.
     def new_post_foorm
       new_general_foorm(survey_names: POST_SURVEY_CONFIG_PATHS, day: nil)
```

Comment:
maybe a follow-up but now this method just wraps another method. Is it worth just deleting this method?

Reply:
Oh true! I'm already making the update to fix the quotes so I'll fix this too!